examples/update_git_hub_labels.py

```diff
@@ -0,0 +1,105 @@
+from typing import List, Dict
+
+from github import Github, Label as GitHubLabel
+from github.Repository import Repository as GitHubRepository
+
+from common.git_hub_api import get_client, submit_graphql_query
+from common.labels import Label, get_labels
+from common.repos import Repo, get_repos
+
+
```

Comment:
Can you add some documentation in the readme about the ability to run the `update_git_hub_labels.py` locally?

Reply:
It's the same as what's listed here: https://github.com/sendgrid/dx-automator#usage---standalone-scripts

No args are needed to run the script.